opencti-platform/opencti-graphql/src/modules/playbook/playbook-components.ts

```diff
@@ -110,7 +110,7 @@ const PLAYBOOK_LOGGER_COMPONENT_SCHEMA: JSONSchemaType<LoggerConfiguration> = {
       oneOf: [
         { const: 'debug', title: 'debug' },
         { const: 'info', title: 'info' },
-        { const: 'warning', title: 'warning' },
+        { const: 'warn', title: 'warn' },
```

Comment:
Seems ok but I don't know how to test this part?

Reply:
you need a component 
![image](https://github.com/user-attachments/assets/8adb87a4-07e9-43ca-8f99-6c4a6fb4094b)

if you test on master with 'warning', you will see an error in console log of opencti, when using this fix the bundle is in log with warn level.

Reply:
tested ok 👌